eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.js

```diff
@@ -45,6 +45,24 @@ import {
   buildShortcutsPath,
 } from "utils/UrlUtils";
 
+const PlatformGCP = styled.h3`
+  color: ${colors.white};
+  background: ${colors.oceanBlue};
+  position: relative;
+  text-align: center;
+  padding: 0.5em;
+  margin-top: 0;
+`;
+
+const PlatformAWS = styled.h3`
+  color: ${colors.white};
+  background: ${colors.blue};
+  position: relative;
+  text-align: center;
+  padding: 0.5em;
+  margin-top: 0;
+`;
+
```

Comment:
We could refactor this into a reusable component file, to improve maintainability if changes are required in the future. The styling applied is the same for both GCP and AWS, other than background.

```
const PlatformGCP = styled.h3`
  color: ${colors.white};
  background: ${colors.oceanBlue};
  position: relative;
  text-align: center;
  padding: 0.5em;
  margin-top: 0;
`;

const PlatformAWS = styled.h3`
  color: ${colors.white};
  background: ${colors.blue};
  position: relative;
  text-align: center;
  padding: 0.5em;
  margin-top: 0;
`;
```

An example of how the code may look in a component file:
```
const Platform = styled.h3`
  color: ${colors.white};
  background: ${enableOn(["gcp"]) ? colors.oceanBlue : colors.blue};
  position: relative;
  text-align: center;
  padding: 0.5em;
  margin-top: 0;
`;
```

Reply:
But the content is different so I took the hit